dev/com.ibm.ws.security.jwt_fat.builder/fat/src/com/ibm/ws/security/jwt/fat/builder/utils/BuilderHelpers.java

```diff
@@ -52,13 +52,13 @@ public class BuilderHelpers {
 
     public static JSONObject setDefaultClaims(LibertyServer server) throws Exception {
         JSONObject settings = new JSONObject();
-        settings.put(PayloadConstants.ISSUER, SecurityFatHttpUtils.getServerIpSecureUrlBase(server) + "jwt/defaultJWT");
+        settings.put(PayloadConstants.ISSUER, "testIssuer");
         return setDefaultClaims(settings);
     }
 
     public static JSONObject setDefaultClaims(String builderId) throws Exception {
         JSONObject settings = new JSONObject();
-        settings.put(PayloadConstants.ISSUER, builderId);
+        settings.put(PayloadConstants.ISSUER, "testIssuer");
         return setDefaultClaims(settings);
     }
```

Comment:
These are now the exact same method, and neither of them need a method argument.

Reply:
I'll remove the first put in another pull. (as usual, another cut/paste error)  - you'll need to re-review.